asab/api/service.py

```diff
@@ -110,4 +113,21 @@ def _initialize_zookeeper(self):
 
 		#get zookeeper-serivice
 		zksvc = self.App.get_service("asab.ZooKeeperService")
-		return zksvc.build_container()
\ No newline at end of file
+		return zksvc.build_container()
+
+	def changelog(self, request):
+		try:
```

Comment:
try/except blocks should not be used to create functionality, please try to use os module for this, something like os.path.isfile()